docs/3.0.0-beta.x/concepts/models.md

```diff
@@ -177,6 +177,18 @@ The options key on the model-json states.
 }
 ```
 
+- `populateCreatorFields`: Configure whether or not the API response should include `created_by` and `updated_by` fields.
```

Comment:
Don't apply this to the beta docs since the feature doesn't exist here

Reply:
Nice catch, thanks 🙂